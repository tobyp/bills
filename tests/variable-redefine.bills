#!./bills
%set x 2
A	B.	5*x

%set x 5
B	A.	2*x
