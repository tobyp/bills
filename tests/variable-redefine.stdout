todo: A pays 0.0 to B
